specs-go/config.go

```diff
@@ -613,6 +613,13 @@ type Arch string
 // LinuxSeccompFlag is a flag to pass to seccomp(2).
 type LinuxSeccompFlag string
 
+// LinuxSeccompFlag options
+const (
+	FlagTSync LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_TSYNC"
```

Comment:
I'm not sure we want to add a const for this. This flag should probably not be used, as we asked here: https://github.com/opencontainers/runtime-spec/issues/1077.

Maybe it is better to _not_ facilitate using it, or even better mark it as deprecated in the spec? (the latter out of scope for this PR, but a follow-up PR to discuss it would be great IMHO).

Reply:
Good point; I'm fine with removing this one; I wasn't aware of the other discussion 👍